index.js

```diff
@@ -2,53 +2,94 @@
 //Task a: declare a variable called votingAge, console log true if age > 18 (no function required)
 
 
+const votingAge = 38;
 
+if (votingAge > 18) {
+  console.log('true');
+}
```

Comment:
The condition is valid, just be sure to always add the "catch-all", which would just be the `else` 
```
if (votingAge > 18) {
  console.log(true)
} else {
  console.log(false)
}
```